include/ofi_mr.h

```diff
@@ -427,9 +438,9 @@ int ofi_mr_cache_search(struct ofi_mr_cache *cache,
  *				with the cache.
  */
 struct ofi_mr_entry *ofi_mr_cache_find(struct ofi_mr_cache *cache,
-				       const struct fi_mr_attr *attr);
+				       const struct fi_mr_attr *attr, uint64_t flags);
```

Comment:
Line is too long. Break after the comma.

Reply:
fixed